src/extensions/urls.ts

```diff
@@ -236,11 +238,65 @@ export const ExtensionsUrls = {
     ),
 };
 
-type CustomAppDetailsUrlDialog =
-  | "create-token"
-  | "remove-webhook"
-  | "remove-token"
-  | "app-activate"
-  | "app-deactivate";
+export const AppSections = {
+  appsSection: "/apps/",
+};
+
+export const AppPaths = {
+  appListPath: AppSections.appsSection,
+  resolveAppPath: (id: string) => urlJoin(AppSections.appsSection, id, "app"),
+  appInstallPath: urlJoin(AppSections.appsSection, "install"),
+};
+
+export const AppUrls = {
```

Comment:
issue: I think we have to remove this old `AppUrls` since it uses old routes, which are no longer rendered in our `index.tsx`. Imho we should get rid of these methods altogether and use ones from `ExtensionsUrls`